fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryScanPlan.java

```diff
@@ -112,20 +144,17 @@ public boolean hasLoadBykeys() {
     @Nonnull
     @Override
     public String toString() {
-        String range;
-        range = getRange();
+        final String range = getTupleRangeOptional().map(TupleRange::toString).orElse(comparisons.toString());
         return "Scan(" + range + ")";
     }
 
     @Nonnull
-    private String getRange() {
-        String range;
+    private Optional<TupleRange> getTupleRangeOptional() {
```

Comment:
Same comment about using a nullable `TupleRange` instead of an `Optional<TupleRange>`.

Reply:
done.